bundle/pom.xml

```diff
@@ -6,7 +6,7 @@
   <parent>
     <groupId>org.neo4j.driver</groupId>
     <artifactId>neo4j-java-driver-parent</artifactId>
-    <version>5.8-SNAPSHOT</version>
+    <version>5.7-SNAPSHOT</version>
```

Comment:
Is this still valid if we are releasing in 5.8?

Reply:
It is fine, we will have a chance to update it later.